support-frontend/assets/pages/showcase/components/hero.tsx

```diff
@@ -13,13 +14,18 @@ import HeroImg from './hero.svg';
 import { regionalContent } from './regionalContent';
 import './hero.scss';
 
-const Caption = (props: { className: string; captionText: string }) => (
-	<div className={props.className}>
-		<figcaption className="showcase-hero__caption">
-			<p>{props.captionText}</p>
-		</figcaption>
-	</div>
-);
+function Caption(props: {
+	className: string;
+	captionText: EmotionJSX.Element;
```

Comment:
Captions are stored in [`regionalContent.tsx`](https://github.com/guardian/support-frontend/blob/ee1e90f7f83fb911f02238445ae35da568e67dce/support-frontend/assets/pages/showcase/components/regionalContent.tsx) as JSX Elements, not as strings. The current support showcase has a <span> element inside the <p> rendered here, so this change seems consistent with that.

<img width="1582" alt="image" src="https://user-images.githubusercontent.com/11380557/154265694-746078f8-6891-4b15-a9ca-c4df8012f73d.png">

I suppose the alternative would be to rejig captions in `regionalContent.tsx` to be actual strings?